scripts/run_docker_build.sh

```diff
@@ -13,6 +13,8 @@ channels:
 
 always_yes: true
 show_channel_urls: true
+track_features: 
+ - nomkl
```

Comment:
This is not OK as it will change the state for all recipes in staged recipes. Can you do that in the recipe?